dist/article.html

```diff
@@ -4,11 +4,12 @@
   <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" />
```

Comment:
dist folder would be used only on ```gh-pages``` to publish, but we can remove from here and publish on that branch later.